src/server/session.ts

```diff
@@ -601,7 +601,7 @@ namespace ts.server {
 
             const definitions = project.getLanguageService().getDefinitionAtPosition(file, position);
             if (!definitions) {
-                return emptyArray;
+                return undefined;

```

Comment:
>undefined [](start = 23, length = 9)

This was probably fine as `emptyArray`, but I have no problem with reverting it.